bin/citrea/provers/risc0/guest-bitcoin/Cargo.toml

```diff
@@ -28,17 +28,26 @@ secp256k1_v028 = { package = "secp256k1", version = "0.28", git = "https://githu
 k256 = { package = "k256", version = "0.13.3", git = "https://github.com/risc0/RustCrypto-elliptic-curves", tag = "k256/v0.13.3-risczero.0" }
 
 [profile.dev]
+debug = 0
+lto = true
 opt-level = 3
+codegen-units = 1
 
 [profile.dev.build-override]
```

Comment:
Only for build scripts, proc macros, and their dependencies. Does not do anything for the binary itself.